src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -286,6 +286,24 @@ protected function sortTaggedServicesByPriority(array $services): array
         return array_keys($maxPriority);
     }
 
+    protected function sortTagsByPriority(array $tags): array
+    {
+        foreach ($tags as &$tag) {
```

Comment:
I would not use a reference here, but create a new array to be returned instead.